R/finalize_explanation.R

```diff
@@ -153,3 +168,238 @@ compute_shapley_new <- function(internal, dt_vS) {
 
   return(dt_kshap)
 }
+
+
+
+#' Mean Squared Error of the Contribution Function `v(S)`
```

Comment:
General comment: Try to shorten the code a bit. Comments are good for getting an overview of what is done, but we don't need it for every single line (normalizing + extracting internal parameters are examples of obvious lines).